CesiumGltfReader/src/GltfReader.cpp

```diff
@@ -639,39 +640,71 @@ ImageReaderResult GltfReader::readImage(
   }
 
   {
-    CESIUM_TRACE("Decode JPG / PNG");
-
-    image.bytesPerChannel = 1;
-    image.channels = 4;
-
-    int channelsInFile;
-    stbi_uc* pImage = stbi_load_from_memory(
-        reinterpret_cast<const stbi_uc*>(data.data()),
-        static_cast<int>(data.size()),
-        &image.width,
-        &image.height,
-        &channelsInFile,
-        image.channels);
-    if (pImage) {
-      CESIUM_TRACE(
-          "copy image " + std::to_string(image.width) + "x" +
-          std::to_string(image.height) + "x" + std::to_string(image.channels) +
-          "x" + std::to_string(image.bytesPerChannel));
-      // std::uint8_t is not implicitly convertible to std::byte, so we must use
-      // reinterpret_cast to (safely) force the conversion.
+
+    tjhandle tjInstance = tjInitDecompress();
+    int inSubsamp, inColorspace;
+    if (!tjDecompressHeader3(
+            tjInstance,
+            reinterpret_cast<const unsigned char*>(data.data()),
+            static_cast<unsigned long>(data.size()),
+            &image.width,
+            &image.height,
+            &inSubsamp,
+            &inColorspace)) {
+      CESIUM_TRACE("Decode JPG");
+      image.bytesPerChannel = 1;
+      image.channels = 4;
       const auto lastByte =
           image.width * image.height * image.channels * image.bytesPerChannel;
       image.pixelData.resize(static_cast<std::size_t>(lastByte));
-      std::uint8_t* u8Pointer =
-          reinterpret_cast<std::uint8_t*>(image.pixelData.data());
-      std::copy(pImage, pImage + lastByte, u8Pointer);
-      stbi_image_free(pImage);
+      if (tjDecompress2(
+              tjInstance,
+              reinterpret_cast<const unsigned char*>(data.data()),
+              static_cast<unsigned long>(data.size()),
+              reinterpret_cast<unsigned char*>(image.pixelData.data()),
+              image.width,
+              0,
+              image.height,
+              TJPF_RGBX,
```

Comment:
The docs says `TJPF_RGBX` leaves the alpha component undefined. That seems like a pretty big problem, unless I'm missing something.
https://rawcdn.githack.com/libjpeg-turbo/libjpeg-turbo/1.5.x/doc/html/group___turbo_j_p_e_g.html#gac916144e26c3817ac514e64ae5d12e2a